docs/source/bmi.control_funcs.rst

```diff
@@ -26,6 +26,12 @@ Initialization
     /* C++ */
     void Initialize (const char *config_file);
 
+.. code-block:: fortran
+
+   ! Fortran (>=2003)
+   integer function initialize(self, config_file)
+
+
```

Comment:
Sure! I can do this. Do you think it'll be too verbose, though?

Reply:
It will be a little verbose but there's not too much we can do about it - Fortran's just a verbose language. The extra information is important, I think.

Reply:
Alright, will do.

Reply:
I'm not sure I like this. It's more analogous to the function prototypes in C/C++, but it's so verbose I worry it may be confusing. I'd rather keep these docs clean, and have the user see these details in the actual code.

Reply:
OK, the compact Fortran function form looks better.

Reply:
I'm all done. If you think it's OK, feel free to merge it, @mcflugen.